patches/0002-Add-crypto-backend-GOEXPERIMENTs.patch

```diff
@@ -387,10 +387,38 @@ index 00000000000000..eb8a026982259c
 +
 +package main
 diff --git a/src/internal/buildcfg/exp.go b/src/internal/buildcfg/exp.go
-index 707776b374b0d4..91b4a71e1ee836 100644
+index 707776b374b0d4..6eab28497a3f2e 100644
 --- a/src/internal/buildcfg/exp.go
 +++ b/src/internal/buildcfg/exp.go
-@@ -67,6 +67,18 @@ func ParseGOEXPERIMENT(goos, goarch, goexp string) (*ExperimentFlags, error) {
+@@ -6,6 +6,7 @@ package buildcfg
+ 
+ import (
+ 	"fmt"
++	"os"
+ 	"reflect"
+ 	"strings"
+ 
+@@ -29,6 +30,19 @@ type ExperimentFlags struct {
+ // default in the current toolchain. This is, in effect, the "control"
+ // configuration and any variation from this is an experiment.
+ var Experiment ExperimentFlags = func() ExperimentFlags {
++	if os.Getenv("MS_GO_NOSYSTEMCRYPTO") == "1" {
```

Comment:
Hmm, wondering if this needs to be added to `src/cmd/go/internal/envcmd/env.go` or somewhere to invalidate the build cache when it changes. It seems like since all it does is (always) modify `GOEXPERIMENT`, which is on that list, it might be fine.